fineract-provider/build.gradle

```diff
@@ -302,140 +125,6 @@ configurations {
     runtime
 }
 
-tasks.withType(JavaCompile) {
-    options.compilerArgs += [
-        "-Xlint:unchecked",
-        "-Xlint:cast",
-        "-Xlint:auxiliaryclass",
-        "-Xlint:deprecation",
-        "-Xlint:dep-ann",
-        "-Xlint:divzero",
-        "-Xlint:empty",
-        "-Xlint:exports",
-        "-Xlint:fallthrough",
-        "-Xlint:finally",
-        "-Xlint:module",
-        "-Xlint:opens",
-        "-Xlint:options",
-        "-Xlint:overloads",
-        "-Xlint:overrides",
-        "-Xlint:path",
-        "-Xlint:processing",
-        "-Xlint:removal",
-        "-Xlint:requires-automatic",
-        "-Xlint:requires-transitive-automatic",
-        "-Xlint:try",
-        "-Xlint:varargs",
-        "-Xlint:preview",
-        "-Xlint:static",
-        "-Werror"
-    ]
-    // TODO FINERACT-959 (gradually) enable -Xlint:all (see "javac -help -X")
-
-    options.deprecation = true
-    options.errorprone {
-        disableWarningsInGeneratedCode = true
-        //Todo Remove necessary checks from this list
-        disable(
-                "UnusedVariable",
```

Comment:
As part of this refactoring, we seem to have lost this ErrorProne configuration. At least I now get 20 pages of errorprone warnings about all these things that used to be disabled...

Reply:
Thanks for catching that one... will fix that.

Reply:
@ptuomola Found and fixed it... I was a bit too eager when I moved the Java compiler configuration to the root build.gradle. I verified locally, no more Errorprone warnings.